samza-core/src/main/java/org/apache/samza/runtime/RemoteApplicationRunner.java

```diff
@@ -35,12 +37,32 @@
  */
 public class RemoteApplicationRunner extends AbstractApplicationRunner {
 
-  private static final Logger log = LoggerFactory.getLogger(RemoteApplicationRunner.class);
+  private static final Logger LOG = LoggerFactory.getLogger(RemoteApplicationRunner.class);
```

Comment:
Hey, if we are not implementing in the Remote, can we remove the runTask() method in ApplciationRunner so we only add it in LocalApplicationRunner?  Thx